### DISCUSSION TITLE
Images not loading in github readme files

### DISCUSSION THREAD

**Discussion by kalaiselvan369** [community/community#54531] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I was going through this repository's read me file. I could see that images weren't loading. 

https://github.com/android/compose-samples

**Comment by aashay28** [comment]
Hey, it's working perfect. Here is the attached file, please verify it.

[Readme file.pdf](https://github.com/community/community/files/11405580/Readme.file.pdf)

**Comment by airtower-luna** [comment]
It works for me, too. Your profile says you're in India, and those images load from `raw.githubusercontent.com`. Maybe you're affected by this problem?

* #42655

In short, certain providers in India seem to be blocking `raw.githubusercontent.com` at the DNS level, using an alternative DNS server gets around it.
